docs/manual/extendedfeatures.rst

```diff
@@ -140,6 +138,40 @@ Currently, the listview contains all entries on the pages. Often, it is desirabl
                 type: integer
                 label: Pages
 
+------------------------------------
+Group entities in the Navigation Bar
+------------------------------------
+
+Each entity represents and option in the navigation bar at the top. If there are
+too many entities or want to group based on relations between the entities, you
+can add a “navBarGroup” option. In this case we group library and books under
+“Learning Resources”:
+
+.. code-block:: yaml
+
+    library:
+        label: Library
+        table: library
+        navBarGroup: Learning Resources
```

Comment:
This should get an entry within https://github.com/philiplb/CRUDlex/blob/master/docs/manual/crudyamlreference.rst